core/contractHost.js

```diff
@@ -15,6 +15,33 @@ import { makeInviteConfig } from './config/inviteConfig';
 import { makeMint } from './mint';
 import makePromise from '../util/makePromise';
 
+function makeCollect(E, log) {
```

Comment:
Is this just moved up?

Reply:
One minor change: 
```
 const doneP = allSettled(results);	
    Promise.resolve(doneP).then(([wins, refs]) => {
```
became
```
    allSettled(results).then(([wins, refs]) => {
```

Reply:
If it was moved, would it be possible to move it back where it was so this minor change stands out? I'm seeing a wall of addition/subtractions. Is it just git?

Reply:
Yep. done. 

I've seen tools that could do a better job of displaying changes in moved code, but it's not common.

Reply:
looks good!